rest_framework-stubs/fields.pyi

```diff
@@ -514,12 +514,8 @@ class MultipleChoiceField(
         read_only: bool = ...,
         write_only: bool = ...,
         required: bool = ...,
-        default: Union[
-            _Empty, Set[str], Set[int], Callable[[], Set[str]], Callable[[], Set[int]]
-        ] = ...,
-        initial: Union[
-            _Empty, Set[str], Set[int], Callable[[], Set[str]], Callable[[], Set[int]]
-        ] = ...,
+        default: _DefaultInitial[Union[Set[Union[str, int]], Set[str], Set[int]]] = ...,
+        initial: _DefaultInitial[Union[Set[Union[str, int]], Set[str], Set[int]]] = ...,
```

Comment:
Since `Set` is mutable, `Set[str]` is not compatible with `Set[Union[str, int]]`. So I explicitly allowed all three combinations:
* `Set[Union[str, int]]`
* `Set[str]`
* `Set[int]`